### ISSUE TITLE
Virtual environment installation issues

### ISSUE THREAD

**Issue by kz258852** [huang-yh/SelfOcc#14]
Hello author,
I often encounter version matching issues with the Openxlab library when installing dependencies for this project, especially when installing sdfStudio. Is there a solution? Thank you. 

for example,
openxlab 0.0.34 has requirement requests~=2.28.2, but you have requests 2.31.0.
openxlab 0.0.34 has requirement rich~=13.4.2, but you have rich 12.5.1.

**Comment by JimPlayboy** [comment]
downgrade the version of packages will help.
pip install openxlab==0.0.11 
pip install scann=1.2.6

**Comment by huang-yh** [comment]
Yes, we are aware of that. But in most cases, you can just ignore these version mismatching warnings.
